### ISSUE TITLE
Unity projects are not working

### ISSUE THREAD

**Issue by YGanLab** [CapsuleEndoscope/VirtualCapsuleEndoscopy#12]
Hey,

I install the unity, the VRCaps, and the sofaapapi3d packages. However, the VR-Caps-Unity and VR-Caps-Unity-RL are not working. Please see the attached images. Could you please give me any hints on how to make the project work? Thanks!
![VR-Caps-UnityErrorCode](https://github.com/CapsuleEndoscope/VirtualCapsuleEndoscopy/assets/84153607/d51bf1ce-de8e-43d5-a62a-9f0760fbf512)
![VR-Caps-Unity-RLErrorCode](https://github.com/CapsuleEndoscope/VirtualCapsuleEndoscopy/assets/84153607/63644b15-b4e7-4a8d-aea7-ea27a6c200cd)

**Comment by kowyo** [comment]
Hey, I happen to have the same problem few days ago when first time opening the project. I just clone the repository again and open it and it works. (I also installed a blender then but I don't know if it is related to the problem). Anyway, just give it a shot.
PS. Using the same version of unity as this repository does.

**Comment by Edwardwhw** [comment]
hey, I have  the same problem, have you solve this problem?

**Comment by kowyo** [comment]
Yes, I solved it by installing a blender on my device. A specific format could only be opened by Blender.

**Comment by jhiltonjones** [comment]
I also have this problem

**Comment by HsiehTPE** [comment]
I also have this problem

**Comment by wza527** [comment]
I also have this problem，have you solve this problem?
